lib/queuePopulator/QueuePopulator.js

```diff
@@ -87,6 +92,7 @@ const metricsHandler = {
     logReadOffset: wrapGaugeSet(logReadOffsetMetric, defaultLabels),
     logSize: wrapGaugeSet(logSizeMetric, defaultLabels),
     logTimestamp: wrapGaugeSet(logTimestamp, defaultLabels),
+    notificationProcessedEvents,
```

Comment:
the "usage" here is to wrap the metric (using one of the wrapXXXX functions), so that the metricsHandler contains only functions which can be called directly : and instead of `this._metricsStore.notificationProcessedEvents.inc()` you will simply do `this._metricsStore.notificationProcessedEvents()`

we should stick to this, for consistency.